index.js

```diff
@@ -365,7 +353,12 @@ function insertText(textarea: HTMLTextAreaElement, {text, selectionStart, select
   }
 }
 
-function styleSelectedText(textarea: HTMLTextAreaElement, styleArgs: StyleArgs) {
+function isEmail(text) {
+  const regExp = new RegExp(/\S+@\S+\.\S+/)
```

Comment:
I see we've gone for a really simple email check here, which seems fine. I wonder though if we want to be checking that the string isn't prefixed with http(s) so that it's not actually a url. I just notice that any URL with a login in it will get recognised as an email. e.g. http://username:password@example.com

Reply:
Good point! I've thrown together a regex that deals with strings prefixed with `http(s)://`.
```javascript
^(?!http(s)*:\/\/)\S+@\S+\.\S+
```